### PR TITLE
feat(ReportsHours): add new component with new endpoint

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add new service method to get data with the new datahub endpoint.
- Add new service method in double to work local.
- Add tests in service.
- Rename old component to mantain working 2 components.
- Add the new component with the new endpoint.
- Add tests component

### CODE REVIEW

File: src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      >
+        <DopplerIntlProvider locale="en">
+          <ReportsHoursVisits domainName={domainName} dateFrom={dateFrom} dateTo={dateTo} />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.loading-box')).toBeInTheDocument();
+    await waitFor(() => expect(getByText('593')));
```
Comment (cbernat) [inline src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js]: Perhaps this `593` could be in a variable.

File: src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      >
+        <DopplerIntlProvider locale="en">
+          <ReportsHoursVisits domainName={domainName} dateFrom={dateFrom} dateTo={dateTo} />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.loading-box')).toBeInTheDocument();
+    await waitFor(() => expect(getByText('50')));
```
Comment (cbernat) [inline src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js:160]: The same with this `50`

File: src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.test.js]: we have before this discussion and says the both options are ok

**Pull request merged**